libr/bin/blang.c

```diff
@@ -40,6 +41,13 @@ static bool check_swift(RBinSymbol *sym) {
 	return false;
 }
 
+static bool check_golang(RBinSymbol *sym) {
+	if (!strncmp (sym->name, "go.", 3)) {
```

Comment:
just do:
```suggestion
	return !strncmp (sym->name, "go.", 3);
```

Reply:
I will update this accordingly. Eventually, we could do the same type of update for other similar checks in this file as well.

Reply:
Yeah, absolutely. If you are up for it, feel free to improve those other cases as well ;)

Reply:
for sure! I will submit as a separate PR later this week